vtk.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pry', '~> 0.13.0'
   spec.add_development_dependency 'rake', '~> 13.0.0'
   spec.add_development_dependency 'rspec', '~> 3.10.0'
-  spec.add_development_dependency 'rubocop', '~> 1.6.0'
+  spec.add_development_dependency 'rubocop', '~> 1.8.0'
```

Comment:
Probably a nitpick, but since this change is unrelated, maybe we break this out in future PRs. 😅